debezium-core/src/main/java/io/debezium/transforms/debeziumtimestampcoverter/README.md

```diff
@@ -0,0 +1,51 @@
+SMT - Debezium Timestamp Converter:(Extended Version of SMT timestampConverter from connect, supporting debezium or others).
+Also works for any kind of nested org.apache.kafka.connect.data.Struct data.
+
+1. Connect config's:
+	Would split the usage into two parts,
+
+	a.Straight fields, like ts_ms in debezium.
+
+		field – the field name (optional, can be left out in case of primitive data)
+		target.type – desired type (i.e. string, long, Date, Time, Timestamp)
+		format – in case converting to or from a string, a SimpleDateFormat-compatible format string
+
+		example:
+		"transforms":"TimestampConv"
+		"transforms.TimestampConv.type":"org.telmate.SMT.TimestampConverter$Value"
+		"transforms.TimestampConv.field":"ts_ms"
+		"transforms.TimestampConvtarget.type":"Timestamp"
+		"transforms.TimestampConv.timestamp.format":"yyyy-MM-dd hh:mm:ss"
+		"transforms.TimestampConv.date.format":"yyyy-MM-dd"
+
+
+	b.Nested fields, like after or before.
+
+		struct.field - the field which is of Struct type(has nested data)
+		field.type - is a comma seperated string, providing an option to add multiple converters.
+					<Type of Timestamp Object> -> <To Target type>
+					can add multiple type converters, <Type of Timestamp Object> -> <To Target type>,<Type of Timestamp Object> -> <To Target type>
+
+
+		example:
+		"transforms.TimestampConv.struct.field":"after"
+		"transforms.TimestampConv.field.type":"io.debezium.time.Timestamp->string,io.debezium.time.Date->string"
```

Comment:
Why is it that the fields are selected using the type here? What if one would like to transform two fields of the same type differently? I think the best would be to allow for one or more "property paths" to be given, e.g. like so:

    "transforms.TimestampConv.fields" : "payload.after.myfield1,payload.after.myfield2,payload.before.myfield1"
    "transforms.TimestampConv.targettypes" : "string,string,unix"

WDYT?

Reply:
@gunnarmorling, I like your suggestion.

Reply:
or, maybe easier still, would be something like:

```
"transforms.TimestampConv.fields" : "payload.after.myfield1(string),payload.after.myfield2(string), payload.before.myfield1(UNIX)"
```
Note that the typestamp type should not be case sensitive, and whitespace around the `,` delimiter should be allowed, too.

Reply:
@gunnarmorling/@rhauch , i like your approach of specifying the field converter in the fields as below,
"transforms.TimestampConv.fields" : "payload.after.myfield1(string),payload.after.myfield2(string), payload.before.myfield1(UNIX)".

Will make the changes accordingly.

Reply:
I think this could be added separately in a follow-up task. I.e. let's first start with the field-based configuration, as it provides the full flexibility. And if needed, we can later look into providing more abstract means of configuration based on types.